src/Consolonia.Core/Helpers/InputProcessing/PasteBlockMatcher.cs

```diff
@@ -2,6 +2,6 @@
 
 namespace Consolonia.Core.Helpers.InputProcessing
 {
-    public class PasteBlockMatcher<T>(Action<string> onComplete, Func<T, char> toChar)
-        : StartsEndsWithMatcher<T>(onComplete, toChar, @"[200~", @"[201~");
+    public class PasteBlockMatcher<T>(Action<string> onComplete, Func<T, string> toText)
```

Comment:
Should be Rune rather than string? I thought why do we need char or string anyway? or otherwise char1,chat2 could be better?

Reply:
ToUTF16() might be more accurate name.

Reply:
It feels a bit weird to have to pass a callback for this at all.

Reply:
no, I mean different. UTF32 can be still passed over the stack. While strings will be always operated from the heap. I'm just thinking, that if we need two chars, lets replace it with two chars or whatever types is enough. Because string gives possibility to store many bytes which looks like we don't need and same time it bring the heap to operation. On other hand why to bother - it's just input. you decide to resolve this thread please

Reply:
I ended up going with Func<T, Rune>(). It turns out that a single char.ConvertToUTF16() internally simply does a new Rune(cp). Append, AppendLine, etc all will accept a rune, so it makes it clear that we are dealing with a rune.